src/scss/tailwind.css

```diff
@@ -0,0 +1,43 @@
+@tailwind components;
+@tailwind utilities;
+
+:root {
+  --color-primary: 23 93 220;
+  --color-secondary: 206 212 218;
+  --color-success: 0 166 90;
+  --color-danger: 221 75 57;
+  --color-warning: 191 126 22;
+  --color-info: 85 85 85;
+
+  --color-text-body: 33 37 41;
+  --color-text-muted: 108 117 125;
+  --color-text-black50: 0 0 0 0.5;
+  --color-text-white50: 255 255 255 0.5;
+
+  --color-primary-btn-hover: #134eb9;
+  --color-primary-btn-text: #fff;
+
+  --color-secondary-btn-hover: #b8c1ca;
+  --color-secondary-btn-text: #212529;
```

Comment:
Tailwind doesn't natively support themes but rather uses variants, primarily `dark:` to specify how a component should look in dark mode. As we want to be able to support additional themes beyond light and dark, we need to extend the theming which is easiest done by using css variables.